angular.json

```
@@ -3,6 +3,13 @@
   "version": 2,
   "projects": {
     "cook-it": "apps/cook-it",
-    "cook-it-e2e": "apps/cook-it-e2e"
+    "cook-it-e2e": "apps/cook-it-e2e",
+    "feature-recipe-list": "libs/feature-recipe-list",
+    "recipies": "libs/recipies",
+    "shell": "libs/shell"
+  },
+  "@shematics/angular:component": {
+    "changeDetection": "onPush",
+    "style": "scss"
```

if you're not using ngneat/aim generator you should use stadalone
```suggestion
    "style": "scss",
    "standalone": true
```